DigitalLearningSolutions.Web/Views/MyAccount/Index.cshtml

```diff
@@ -92,6 +92,49 @@
             </dd>
           </div>
 
+          @if (Model.DelegateNumber != null)
+          {
+            <div class="nhsuk-summary-list__row account-detailslist_row">
+              <dt class="nhsuk-summary-list__key">
+                Job group:
+              </dt>
+              <dd class="nhsuk-summary-list__value">
+                @Model.JobGroup
+              </dd>
+            </div>
+
+            @if (Model.CustomField1 != null)
+            {
+              <partial name="MyDetailsCustomField" model="@Model.CustomField1" />

```

Comment:
I'm wondering if it would be nicer to just pass in an array of CustomFields/CustomPrompts and use foreach here rather than hardcoding CustomField1, CustomField2 etc (we can filter out the null values)?